src/views/Product.vue

```diff
@@ -91,12 +91,10 @@ export default defineComponent({
     }
   },
   mounted () {
-    this.store.dispatch("job/fetchJobs", {
-      "inputFields":{
-        "systemJobEnumId": Object.values(this.jobEnums),
-        "systemJobEnumId_op": "in"
-      }
-    });
+    emitter.on("selectedShop", this.fetchJobs);
```

Comment:
As you have removed the fetchJobs action from mounted hook, then when the component is mounted we won't get the job data.

Reply:
OK Sir I Will correct it.